.trivyignore

```diff
@@ -9,37 +9,22 @@ CVE-2021-38561
 CVE-2021-44716
 CVE-2022-21698
 CVE-2022-27191
-# kube-state-metrics
-CVE-2021-43565
-CVE-2022-27664
-# /bin/node_exporter
-CVE-2021-44716
+
 # opt/telegraf/telegraf
 CVE-2022-27664
 CVE-2022-29190
 CVE-2022-29222
 CVE-2022-41717
```

Comment:
Thanks Soham, please also check if upgrading telegraf removes some of the telegraf vulnerabilities

Reply:
Thanks Grace! I have pushed the latest commit to update the telegraf CVEs.